readthedocs/api/v2/views/integrations.py

```diff
@@ -159,6 +159,7 @@ def get_integration(self):
             integration = Integration.objects.create(
                 project=self.project,
                 integration_type=self.integration_type,
+                provider_data={},
```

Comment:
Why do we need to add this? Won't integrations already exist without `provider_data`?

Reply:
For some reason tests were failing with provider_data being a `str` type when actually it's a jsonfield :man_shrugging: . I guess a bug in jsonfield, or they need to reload the object from the db to get it working.

Reply:
The `JSONField` we are using is just a `TextField`. So, if we want to make it always a valid JSON, we may want to add `default=dict()` in the db field.

Reply:
Added a default on the model